Source/IGListSectionController.h

```diff
@@ -107,8 +107,27 @@ NS_SWIFT_NAME(ListSectionController)
  @note The default implementation does nothing. **Calling super is not required.**
  */
 - (void)didUnhighlightItemAtIndex:(NSInteger)index;
+    
+/**
+ Identifies whether an object can be moved through interactive reordering.
+ 
+ @param index The index of the unhighlighted cell.
+ 
+ @note Interactive reordering is supported both for items within a single section, as well as for reordering sections themselves when sections contain only one item. The default implementation returns false.
```

Comment:
nit: let's newline. I try to wrap comments at 120 chars (we should add a lint)

Also "false" should be "NO" with markdown code ticks

<img width="646" alt="screen shot 2017-10-24 at 3 57 04 pm" src="https://user-images.githubusercontent.com/739696/31965028-01e7bf40-b8d4-11e7-8333-9876bd6c699f.png">